src/chttpd/src/chttpd_httpd_handlers.erl

```diff
@@ -43,3 +44,6 @@ design_handler(<<"_update">>)  -> fun chttpd_show:handle_doc_update_req/3;
 design_handler(<<"_info">>)    -> fun chttpd_db:handle_design_info_req/3;
 design_handler(<<"_rewrite">>) -> fun chttpd_rewrite:handle_rewrite_req/3;
 design_handler(_) -> no_match.
+
+partition_design_handler(<<"_view">>) -> fun chttpd_view:handle_partition_view_req/4;
+partition_design_handler(_) -> no_match.
```

Comment:
I didn't test it yet, but I am pretty sure we need `partition_handler(_) -> no_match` here (and export it as well).

Reply:
we've run our test suite and manual testing without needing that, what fails without it?